inlong-sort/sort-connectors/elasticsearch-base/src/main/java/org/apache/inlong/sort/elasticsearch/table/ElasticsearchOptions.java

```diff
@@ -155,8 +144,23 @@ public enum BackOffType {
                     .withDescription(
                             "The format must produce a valid JSON document. "
                                     + "Please refer to the documentation on formats for more details.");
+    public static final ConfigOption<String> INLONG_METRIC =
```

Comment:
inlong metric is defined in inlongMetric interface, can just use it

Reply:
DONE